store/copr/mpp.go

```diff
@@ -227,6 +229,25 @@ func (m *mppIterator) handleDispatchReq(ctx context.Context, bo *tikv.Backoffer,
 	m.establishMPPConns(bo, req, taskMeta)
 }
 
+// TODO: retry once failed?
+func (m *mppIterator) cancelMppTask(bo *tikv.Backoffer, meta *mpp.TaskMeta) {
```

Comment:
should this function at least return a true or false on success or failure?

Reply:
as comment in the NOTE.